src/js/resize.js

```diff
@@ -0,0 +1,32 @@
+import Split from 'split.js'
+
+const PAGE_WIDTH = 960
+
+const isSplited = () => document.getElementsByClassName('gutter').length > 0
+const isBigScreen = () => window.innerWidth > PAGE_WIDTH
+const needGutter = () => isBigScreen() && !isSplited()
+const noNeedGutter = () => !isBigScreen() && isSplited()
+
+const main = document.getElementById('main')
+const gutter = () => document.getElementsByClassName('gutter')[0]
```

Comment:
Use global variable `let GUTTER = undefined` and set in `makeGutter` once instead of querying it always